cmd/start_windows.go

```diff
@@ -56,3 +66,41 @@ func startApp(ctx context.Context, client *api.Client) error {
 	}
 	return waitForServer(ctx, client)
 }
+
+func isProcRunning(procName string) []uint32 {
+	pids := make([]uint32, 2048)
+	var ret uint32
+	if err := windows.EnumProcesses(pids, &ret); err != nil || ret == 0 {
```

Comment:
Unlikely but what if there are more than 2048 pids running?

Reply:
Based on the [Win32 API docs](https://learn.microsoft.com/en-us/windows/win32/api/psapi/nf-psapi-enumprocesses#remarks), it looks like there's no way to known up front how large an array you need, it will just return up to the number you ask for, and wont fail if there are more.  So if there were > 2048, and the installer was past the 2048'th entry, we would incorrectly think it's not running (effectively same behavior as today.)  We could add logic to detect if there are 2048 entries, and if so, retry with a larger array.   Given the failure mode is relatively benign, I'm inclined to keep it simpler.